.env.tpl

```diff
@@ -51,6 +51,14 @@ ZGW_API_CLIENTID=op://Dimpact/ZAC-.env-$APP_ENV/ZGW/API_CLIENTID
 ZGW_API_SECRET=op://Dimpact/ZAC-.env-$APP_ENV/ZGW/API_SECRET
 ZGW_API_URL_EXTERN=op://Dimpact/ZAC-.env-$APP_ENV/ZGW/API_URL_EXTERN
 
+# -----------------------------------------
+# e2e only environment variables
+# -----------------------------------------
+TEST_USER_1_USERNAME=op://Dimpact/testuser1/username
```

Comment:
I don't really like the fact that we now have two sets of TEST_USER_1 and 2: one for our local and integration test Docker Compose and one for the test environment. These ones I think are for the test environment? I think we should name them more clearly so that we can see which is which.

I though we had specific e2e test user accounts on the test environment btw? I think that would be a lot better.

Reply:
So maybe just name these `E2E_TEST_USER_1_USERNAME` etc ?